addons/forum-search/userscript.js

```diff
@@ -234,7 +234,7 @@ function appendSearch(box, query, page, term, msg) {
 }
 
 export default async function ({ addon, global, console, msg }) {
-  await addon.tab.loadScript(addon.self.lib + "/thirdparty/cs/scratchblocks-v3.5.1-min.js");
```

Comment:
Just remove the version number, then you don't keep having to change files.

Reply:
That's fine